src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/DateFormatTest.java

```diff
@@ -86,7 +88,7 @@ public void greaterThan() {
     @Test
     public void greaterThanOrEqualTo() {
         assertThat(
-            query(SELECT_FROM + "WHERE date_format(insert_time, 'YYYY-MM-dd', 'UTC') >= '2014-08-23' LIMIT 1000"),
+            query(SELECT_FROM + "WHERE date_format(insert_time, 'yyyy-MM-dd', 'UTC') >= '2014-08-23' LIMIT 2000"),
```

Comment:
Why do you increase the limits ?

Reply:
The return hits in ES 7.0 is different with ES 6.7. ES 7.0 return value doesn't include "2014-08-24" is limited 1000.

Reply:
But is it the expected behavior? I am ok with updating the test queries as long as we know the test is checking the right thing.

Reply:
Yes, it is the expected behavior. In the test data set, there are 1424 records has insert_time 2014-08-23. With the limit 1000, in ES 6.7, the return hits include record with insert_time as 2014-08-23 and 2014-08-24. But in ES 7.0, with the limit 1000, the result only include insert_time 2014-08-23. I didn't find the doc which mention it, but it seems the ES 7.0 sort the output in order.